tracer/test/Datadog.Trace.TestHelpers.AutoInstrumentation/Containers/AerospikeFixture.cs

```diff
@@ -29,6 +29,7 @@ protected override async Task InitializeResources(Action<string, object> registe
         var container = new ContainerBuilder()
                        .WithImage("aerospike/aerospike-server:6.2.0.6")
                        .WithPortBinding(3000, true)
+                       .WithWaitStrategy(Wait.ForUnixContainer().UntilPortIsAvailable(3000))
```

Comment:
There's three 3000s in this file now... I feel like we should extract it out (for when this is inevitably copy-pasted later)

Reply:
Sounds reasonable, done